src/main/java/com/softserveinc/ita/pageobjects/HomePage.java

```diff
@@ -56,8 +56,9 @@ public CategoriesPage openDiscountCategory() {
 
         return new CategoriesPage();
     }
+  
+   public List<String> getTitlesFromListOfLastViewedProducts() {
 
```

Comment:
delete this empty line before return, if between return statement and method declaration no other code, empty line not required